app/assets/src/components/views/compare/SamplesHeatmapView/samples_heatmap_view.scss

```diff
@@ -22,13 +22,26 @@
 
     .filterRow {
       margin-top: $space-l;
-      margin-bottom: $space-s;
+      margin-bottom: $space-l;
+      margin-right: 10px !important;
+    }
+
+    .filterElement {
+      display: inline-flex;
```

Comment:
I don't think you need all of this styling. The following seems sufficient:
```
.filterRow {
  display: flex;
}

.filterElement {
  width: 16.66%;
  padding: 0 10px;
}
```

(`16.66%` looks a little better than the long decimal. Also, the previous styling seems to have `padding: 0 10px`)

Reply:
I just added "col s2" back & removed .filterElement

Reply:
I left `.filterRow` as is because I added back in `"col s2"` from Materialize.css. If I were to add in the `display.flex` and remove the margin-top and bot, it messes up the structure of filters